runtime/compiler/build/toolcfg/zos-xlc/common.mk

```diff
@@ -94,13 +95,18 @@ CX_FLAGS+=\
     -Wc,"TARGET($(TGTLEVEL))"
 
 # Now we get to do this awesome thing because of EBCDIC
+PRODUCT_INCLUDES:=\
+    $(J9SRC)/zos_zlib/hzc/include \
+    $(PRODUCT_INCLUDES)
+
 CX_FLAGS+=\
     -qnosearch \
     $(patsubst %,-I%,$(PRODUCT_INCLUDES)) \
     -DIBM_ATOE \
     -Wc,"convlit(ISO8859-1)" \
     -qsearch=$(J9SRC)/a2e/headers \
     -qsearch=/usr/include \
+    -qsearch=$(J9SRC)/zos_zlib/hzc/include \
```

Comment:
Looking at the xlc documentation, `search` path is looked for the System include files and as we were statically linking System library, I updated `qsearch` path. This will be needed when we upgrade the build compilers, at that time path will be changed.

Reply:
Right, the review was commenting on the ordering of the search paths. Does it matter? If not then we should place them together:

```
    -qsearch=$(J9SRC)/a2e/headers \
    -qsearch=$(J9SRC)/zos_zlib/hzc/include \
    -qsearch=/usr/include \
```

Reply:
Ohh, order does matter, but we won't have conflict with `/usr/include` and other one.

Reply:
Changed order in https://github.com/eclipse/openj9/commit/79d9f4618ef87643cd4ffe77314559d307bacac2